app/frontend/src/components/forms/manage/TeamManagement.vue

```diff
@@ -582,7 +582,7 @@ export default {
         <v-checkbox-btn
           v-if="!disableRole('submission_reviewer', item, form.userType)"
           key="submission_reviewer"
-          v-model="item.columns.submission_reviewer"
+          v-model="item.submission_reviewer"
```

Comment:
Might not be related to this line of code, but if I go to Team Management, click "Add a New Member" icon, it comes up as IDIR. I click "owner" and owner gets a checkbox as expected. Then switch to BCeID Basic, and when you click "form_submitter" it does not get a checkbox.
<img width="266" alt="image" src="https://github.com/bcgov/common-hosted-form-service/assets/35532993/cfbb76c7-7cde-4130-bb06-8f7cde8ce9d1">

Reply:
Can we add tests for this kind of thing? Or is that something that's better to be done when these go to the Composition API?

Reply:
thanks! it seems like it's not clearing the fields when changing the identity provider, allowing for the selected role "owner" to exist in basic bceid when it should not. i can try to add it in as a test!

Reply:
This should be resolved now with added tests